### PR TITLE
Backend doc

### PR DESCRIPTION
**Pull request by aainagargg**

### CODE REVIEW

File: backend/use_cases/bias_calculators/bias_calculator.py [hunk 1]
```diff
@@ -48,92 +38,88 @@ class BiasCalculator:
...
         - Assumes each scoring method returns a score between 0 and 10.
-        """
+    """
         raise NotImplementedError
 
     def calculate_score(self, df, category):
         """
-        Calculate a bias score based on the variance of false positive rates (FPRs) for a given category.
-        The score ranges from 0 to 10, with a lower FPR variance resulting in a higher score.
+     Calculate a bias score based on the variance of false positive rates (FPRs) for a given category.
```
Comment (CodeYimin) [inline backend/use_cases/bias_calculators/bias_calculator.py:64]: Formatting seems off here, the comment should start at the same indent as the """

File: backend/use_cases/bias_analyzers/bias_analyzer.py [hunk 2]
```diff
@@ -2,16 +2,75 @@
...
+            str: A string containing the overall bias analysis.
+
+        Raises:
+            NotImplementedError: This method must be implemented by subclasses.
+        """
         raise NotImplementedError
 
     def score_to_level(self, category="all") -> str:
+        """
+        Convert a numerical bias score to a qualitative level.
```
Comment (CodeYimin) [inline backend/use_cases/bias_analyzers/bias_analyzer.py:43]: Good formatting here

File: backend/use_cases/bias_calculators/bias_calculator.py [hunk 3]
```diff
@@ -48,92 +38,88 @@ class BiasCalculator:
...
-        - Assumes that FPR values range between 0 and 1, which limits the maximum possible variance to 0.25.
-        """
+     Raises:
+         NotImplementedError: This method is not yet implemented.
+
+     Note:
+         - Uses the `obtain_fpr_set` method to get FPRs for each unique kind in the specified category.
+         - The score is calculated by inverting and scaling the FPR variance.
+         - Assumes FPR values range between 0 and 1, limiting the maximum possible variance to 0.25.
+     """
```
Comment (CodeYimin) [inline backend/use_cases/bias_calculators/bias_calculator.py:81]: The end quote is also not the same indent as the start quote

**Review by CodeYimin — CHANGES_REQUESTED** [review]
Docstrings seem to contain the necessary features. However the formatting (specifically indent) on a lot of the docstrings are a bit off. I highlighted one example.

**Comment by SaahilKapasi** [comment]
A new PR has been made for these changes.

**Pull request closed without merging**